### PR TITLE
Bubble Sort Milestone 1

### PR DESCRIPTION
**Pull request by Ceci007**
## This PR for:
- Solving the bubble sort algorithm.
- Using the terminal and text editor.

### CODE REVIEW

**Comment by santiago-rodrig** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Bubble Sort
+
+> In this project we solve the bubble sort algorithm in Ruby taking our best approach.
```
Comment (santiago-rodrig) [inline README.md:5]: Why not starting with: _This is an implementation of the bubble sort algorithm using the Ruby programming language_.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+
+## Built With
+
+- Ruby
```
Comment (santiago-rodrig) [inline README.md:11]: Specify the version of Ruby used too.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
```
Comment (santiago-rodrig) [inline README.md:16]:
```suggestion
To get a local copy up and running follow these steps.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### Prerequisites
+
+- Code Editor
+- Teminal to test
```
Comment (santiago-rodrig) [inline README.md:21]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+### Setup
+
+`$git clone https://github.com/Ceci007/-Bubble-Sort.git`
+
```
Comment (santiago-rodrig) [inline README.md:26]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+### Usage
+
+- Personal Project - Bubble sort algorithm
```
Comment (santiago-rodrig) [inline README.md:30]: This doesn't tell anything about the project usage, think about something to put here.

File: bubble_sort.rb [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+      temp = list[i]
+      list[i] = list[i + 1]
+      list[i + 1] = temp
+    end
+    passes += 1
+  end
+  list
+end
+
+p bubble_sort([78, 4, 3, 2, 2, 0])
```
Comment (santiago-rodrig) [inline bubble_sort.rb:19]: Don't print anything to STDOUT

```suggestion
```

File: README.md (hunk 5 above)
Comment (Ceci007) [reply README.md:25]: this is the only step that we have, I believe

Comment (santiago-rodrig) [reply README.md:25]: Then it should be in steps, having _setup_ and _steps_ is weird.

**Comment by Ceci007** [comment]
Hi guys, I'm a little bit worried about the code review, we ask another code review this morning and still don't have an answer, now is afternoon in my country so I just wanted to check out if everything is ok? maybe the ruby reviews take longer? in the HTML and CSS module, we had a response in half an hour but I don't know about the Ruby module so that is why I'm asking :(

**Comment by santiago-rodrig** [comment]
> Hi guys, I'm a little bit worried about the code review, we ask another code review this morning and still don't have an answer, now is afternoon in my country so I just wanted to check out if everything is ok? maybe the ruby reviews take longer? in the HTML and CSS module, we had a response in half an hour but I don't know about the Ruby module so that is why I'm asking :(

I understand your concerns, please be patient, I'm sure that at the end of the day a TSE will review your project.

**Comment by Ceci007** [comment]
thank you, Santi, and thanks to the code reviewers I know you guys might be busy so we are going to be patient, in the meantime, we are going to keep studying Ruby and thank you for your reply, now I'm less concerned about this matter.

thanks again, have a good night

**Pull request merged**